### PR TITLE
Added special vlans to Interface events.

### LINKED ISSUE #404: Duplicated `"untagged"` and `"any"` EVCs
`_is_duplicated_evc()` used to find duplicated EVCs but since vlans are allocated it is outdated. Even though it compares every EVC vlan type (string, integer, list), it is only expected to find duplicated EVCs for "untagged" and "any" cases.

### Solution
Have a dictionary in each Interface for available special cases: `self.available_special: {"vlan": {"any", "untagged"}}`, and take values in/out for the usage of these. This dictionary would need to be saved in the DB document `interface_details`.
Changes to kytos repository is also necessary.

##### Thoughts
It can also be done with only changing `mef_eline` by having a dictionary storing only EVCs with "untagged" and "any" VLANs so they could be compared with new inputs either from creation or updates involving these values.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #404 

### Summary
Events for vlan usage now include special vlans "untagged" and "any
Related kytos [PR](https://github.com/kytos-ng/kytos/pull/431)

### Local Tests
Tested with UI and API requests. Created, deleted, modified EVCs.

### End-To-End Tests
```
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.3.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2, anyio-3.6.2
collected 244 items

tests/test_e2e_01_kytos_startup.py ..                                    [  0%]
tests/test_e2e_05_topology.py ..................                         [  8%]
tests/test_e2e_10_mef_eline.py ..........ss.....x.....x................  [ 24%]
tests/test_e2e_11_mef_eline.py ......                                    [ 27%]
tests/test_e2e_12_mef_eline.py .....Xx.                                  [ 30%]
tests/test_e2e_13_mef_eline.py ....Xs.s.....Xs.s.XXxX.xxxx..X.....F..... [ 47%]
.                                                                        [ 47%]
tests/test_e2e_14_mef_eline.py x                                         [ 47%]
tests/test_e2e_15_mef_eline.py ....                                      [ 49%]
tests/test_e2e_20_flow_manager.py .....................                  [ 58%]
tests/test_e2e_21_flow_manager.py ...                                    [ 59%]
tests/test_e2e_22_flow_manager.py ...............                        [ 65%]
tests/test_e2e_23_flow_manager.py ..............                         [ 71%]
tests/test_e2e_30_of_lldp.py ....                                        [ 72%]
tests/test_e2e_31_of_lldp.py ...                                         [ 74%]
tests/test_e2e_32_of_lldp.py ...                                         [ 75%]
tests/test_e2e_40_sdntrace.py .............                              [ 80%]
tests/test_e2e_41_kytos_auth.py ........                                 [ 84%]
tests/test_e2e_42_sdntrace.py ..                                         [ 84%]
tests/test_e2e_50_maintenance.py ........................                [ 94%]
tests/test_e2e_60_of_multi_table.py .....                                [ 96%]
tests/test_e2e_70_kytos_stats.py ........                                [100%]
```

Error from `test_190_post_an_evc_twice()` because the EVC has no tag. Looking into it.
[Issue](https://github.com/kytos-ng/mef_eline/issues/410) created.

### CODE REVIEW

File: requirements/dev.in [hunk 1]
```diff
@@ -5,5 +5,5 @@
 #    pip-compile --output-file requirements/dev.txt requirements/dev.in
 #
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git@epic/vlan_range#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/kytos.git@epic/vlan_range_special#egg=kytos[dev]
```
Comment (Alopalao) [inline requirements/dev.in:8]: Leaving comment as reminder to delete this requirement.

File: main.py [hunk 2]
```diff
@@ -907,6 +881,7 @@ def load_all_evcs(self):
         for circuit_id, circuit in circuits:
             if circuit_id not in self.circuits:
                 self._load_evc(circuit)
+        emit_event(self.controller, "evcs_loaded", content=dict(circuits))
```
Comment (viniarck) [inline main.py:884]: Once you merge `master` into your branch or the targeted branch this diff isn't expected to show up, heads up.

**Review by viniarck — APPROVED** [review]
Looks good. Leaving this pre-approved. Let's make sure to revert, and also merge `master` into this branch when the time comes, before merging this `branch` into `master`.

File: main.py (hunk 2 above)
Comment (Alopalao) [reply main.py:884]: This line added by my commit in this PR, somehow @.@

Comment (Alopalao) [reply main.py:884]: Looks like `dev.in` has a conflict to so going to merge branches when `kytos` and `topology` are ready to land.

Comment (Alopalao) [reply main.py]: Merged master

File: requirements/dev.in (hunk 1 above)
Comment (Alopalao) [reply requirements/dev.in]: Reverted

**Comment by viniarck** [comment]
Aldo has mentioned he'll address issue #410 in a new PR, so we'll keep with this temporary `test_190_post_an_evc_twice` e2e failure in the meantime. Let's ship it.

**Pull request merged**